src/deploy/standalone/noobaa_syslog.conf

```diff
@@ -4,6 +4,17 @@ $umask 0000
 $FileCreateMode 0644
 $EscapeControlCharactersOnReceive off
 
+# Remove defaul ratelimiting, Setting this value to 0 turns off ratelimiting
+$imjournalRatelimitInterval 0
+$imjournalRatelimitBurst 0
+
+# start log rotation via outchannel
+# outchannel definition, 100M limit, If updating this value update logrotate also.
+$outchannel log_rotation,/var/log/noobaa.log,104857600,/usr/local/noobaa-core/src/deploy/standalone/logrotate_noobaa
```

Comment:
It will work without `.sh` also

Reply:
I reverted back to `bash /usr/local/noobaa-core/src/deploy/standalone/logrotate_noobaa.sh` because rsyslog still giving same error even after adding `|| true`
```
May 07 05:33:44 d310af06fbec rsyslogd[1026]: program '/usr/local/noobaa-core/src/deploy/standalone/logrotate_noobaa.sh' (pid 1065) exited with status 1 [v8.2310.0-4.el9]
May 07 05:33:44 d310af06fbec rsyslogd[1026]: file size limit cmd for file '/var/log/noobaa.log' did no resolve situation  [v8.2310.0-4.el9]
```

Reply:
did you chmod +x that script? it doesn't make sense